lib/healthchecks/queue_latency_check.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module HealthChecks
+  class QueueLatencyCheck < OkComputer::Check
+    attr_accessor :threshold
+
+    def initialize(threshold = 30)
+      self.threshold = threshold.to_i
```

Comment:
It's preferable to use `@threshold` instead of `self.threshold` for attr_accessor variables.

Reply:
Thanks for the tip. I've seen it both ways, The OkComputer gem uses `self` _shrug_